docs/source/ecosystem_overview/core.md

```diff
@@ -0,0 +1,168 @@
+# The Core JAX AI Stack
+
+The core JAX AI Stack consists of five key libraries that provide the foundation for model development: JAX, [Flax](https://flax.readthedocs.io/en/stable/), [Optax](https://optax.readthedocs.io/en/latest/), [Orbax](https://orbax.readthedocs.io/en/latest/) and [Grain](https://google-grain.readthedocs.io/en/latest/).
+
+## JAX: A Foundation for Composable, High-Performance Program Transformation
+
+[JAX](https://docs.jax.dev/en/latest/) is a Python library for accelerator-oriented array computation and program transformation, designed for high-performance numerical computing and large-scale Machine Learning. With its functional programming model and friendly, NumPy-like API, JAX provides a solid foundation for higher-level libraries.
+
+With its compiler-first design, JAX inherently promotes scalability by leveraging [XLA](https://openxla.org/xla) (see Section \<\*\*\*\*\*\*\>) for aggressive, whole-program analysis, optimization, and hardware targeting. The JAX emphasis on functional programming (i.e., pure functions) makes its core program transformations more tractable and, crucially, composable
```

Comment:
The `(see Section \<\*\*\*\*\*\*\>)` needs to be replaced by the x-ref to the XLA section in the TR